src/components/Jam.jsx

```diff
@@ -114,7 +55,13 @@ export default function Jam() {
   const [collaborativeOperationRunning, setCollaborativeOperationRunning] = useState(false)
   const [schedule, setSchedule] = useState(null)
 
-  const schedulerPreconditionSummary = useSchedulerPreconditionSummary(walletInfo, SCHEDULER_START_ACCOUNT)
+  const startJarUtxos = useMemo(() => {
+    if (!walletInfo) return null
+
+    return walletInfo.data.utxos.utxos.filter((it) => it.mixdepth === SCHEDULER_START_ACCOUNT)
+  }, [walletInfo])
```

Comment:
Side note, unrelated to this PR: We have to be careful to harmonize terminology now that we have "Jars". We had "Privacy Levels" in the past and there's still "Accounts" and "Mixdepths". All referring to the same thing.

Reply:
Yes, we should make sure to only use one term. Should be easy enough to clean up, at least on the frontend side of things. Will also be important for documentation.